src/pages/prosessmal/[slug].tsx

```diff
@@ -48,29 +24,65 @@ const useStyles = makeStyles({
   },
 });
 
-const ProcessTemplate = ({ processTemplates, employees, professions, tags }: InferGetServerSidePropsType<typeof getServerSideProps>) => {
-  const router = useRouter();
-  const { slug } = router.query;
+export const getServerSideProps: GetServerSideProps = async ({ params }) => {
+  const processTemplate = await prisma.processTemplate.findUnique({
+    where: {
+      slug: params.slug.toString(),
+    },
+    include: {
+      phases: {
+        orderBy: {
+          order: 'asc',
+        },
+        select: {
+          id: true,
+          order: true,
+          title: true,
+          tasks: {
+            where: {
+              global: true,
+            },
+            select: {
+              id: true,
+              title: true,
+              description: true,
+              responsible: {
+                select: {
+                  firstName: true,
+                  lastName: true,
+                  imageUrl: true,
+                },
+              },
+            },
+          },
+        },
+      },
+    },
+  });
+
+  return { props: { processTemplate } };
+};
 
+const ProcessTemplate = ({ processTemplate }: InferGetServerSidePropsType<typeof getServerSideProps>) => {
   const classes = useStyles();
 
-  const processTemplate: IProcessTemplate = processTemplates.find((processTemplate) => processTemplate.slug === slug);
-
   return (
     <>
       <Head>
-        <title>Prosessmal - {slug}</title>
+        <title>Prosessmal - {processTemplate?.title}</title>
```

Comment:
For estetikkens skyld:
```suggestion
        <title>Prosessmal {processTemplate && `- ${processTemplate.title}`}</title>
```